drakrun/drakrun/draksetup.py

```diff
@@ -373,11 +370,7 @@ def install(vcpus, memory, storage_backend, disk_size, iso_path, zfs_tank_name,
 
     cfg_path = os.path.join(VM_CONFIG_DIR, "vm-0.cfg")
 
-    try:
-        subprocess.run('xl create {}'.format(shlex.quote(cfg_path)), shell=True, check=True)
-    except subprocess.CalledProcessError:
-        logging.exception("Failed to launch VM vm-0")
-        return
+    vm0.create(cfg_path)
```

Comment:
We've lost the error message

Reply:
`create` is calling `try_run` internally, and it will give us the error message occuring during the subprocess commands

Reply:
Ah, right :+1: